.gitlab-ci.yml

```diff
@@ -0,0 +1,25 @@
+image: archlinux/base
```

Comment:
Should probably use `archlinux` over `archlinux/base`.

Reply:
because?